services/user.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> get_user_model():

```

Comment:
The return type hint should be `User` (or the appropriate user model class) instead of `get_user_model()`, as `get_user_model()` is a function call, not a type.